aiven/client/client.py

```diff
@@ -22,6 +22,9 @@ def __init__(self, response, status=520):
         self.status = status
 
 
+UNDEFINED = object()
```

Comment:
Just use None for this.

Reply:
There are three options for the `project_vpc_id` property in the HTTP API:
- Setting it to a UUID (a result of using `--project-vpc-id`)
- Setting it to `null` (a result of using `--no-project-vpc`)
- Not defining the property (a result of not using either of the switches)

The `UNDEFINED` constant is used for the third case - a None value leads to the second (`null`), which has a different meaning. This way is backwards compatible.